### PR TITLE
4.0.0-M1 news item and docs

### PR DESCRIPTION
**Pull request by jeenbroekstra**
Javadocs and news item for 4.0.0-M1

### CODE REVIEW

File: site/content/download.md [hunk 1]
```diff
@@ -19,6 +19,22 @@ For details on what’s new and how to upgrade, see the [release and upgrade not
 - [RDF4J artifacts](https://search.maven.org/search?q=org.eclipse.rdf4j) on the [Maven Central Repository](http://search.maven.org/)
 
 
+## RDF4J 4.0.0 Milestone 1
+
+RDF4J 4.0.0 Milestone 1 is the first milestone build of our upcoming 4.0 release. It requires Java 11 minimally.
```
Comment (hmottestad) [inline site/content/download.md:24]: Maybe: Java 11 or later is required.

File: site/content/news/rdf4j-400-M1.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Some of the highlights covered in this first milestone:
+
+- update to Java 11 as the minimally-required version of Java
+- improvements in the SparqlBuilder
+- performance improvements in the SPARQL query engine
+- performance improvements in the SHACL engine
+- improved support for working with RDF4J in Spring and Spring Boot applications
+- SeRQL and SPIN are no longer supported
+- removal of several long-deprecated packages and interfaces
+
```
Comment (hmottestad) [inline site/content/news/rdf4j-400-M1.md:20]: And performance improvements to the MemoryStore.

**Comment by hmottestad** [comment]
Made some comments, otherwise LGTM!

**Comment by jeenbroekstra** [comment]
Thanks @hmottestad, fixed the things you noticed. I've also added in the spring documentation that Florian wrote.

**Pull request merged**